### ISSUE TITLE
Option to put the extention in system menu

### ISSUE THREAD

**Issue by qmarlats** [martin31821/cpupower#29]
Can you add an option to place the extention in top right menu please?

**Comment by martin31821** [comment]
What do you mean by "top right menu" exactly? 
I think it's the status window where volume etc. are controlled?

If so, I can try. 
Do you want the profile selection and/or the sliders / switches?

**Comment by qmarlats** [comment]
Yep it's this menu, sorry I don't know how it's called. ^^ I think you can simply add an option to move the extension in this menu, by replacing the dropdown by the sort of dropdown in the right menu (like this: https://deblinux.files.wordpress.com/2014/06/media-player-indicator.png?w=665). I hope you understood me, I'm not clear at all and I'm French ^^
![capture d ecran de 2016-04-22 22-18-46](https://cloud.githubusercontent.com/assets/3251811/14753878/5259a428-08d8-11e6-9d70-cbc46daa37e9.png)

**Comment by shihrer** [comment]
I think this would be a great option to have.

**Comment by tonylambiris** [comment]
I like the idea, there's an extension I use that does something similar  I'll keep it on the back-burner.

**Comment by KristiaN1337** [comment]
A friend and I have been trying to make this work for the last couple of days, we're not so good, obviously, but could we at least get some pointers on what we'd have to do if this feature will never arrive?

Thanks

**Comment by martin31821** [comment]
Sorry, but I don't know how to achieve this. You might want to have a look at the Bluetooth Quick Connect or Sound Input & Output Device Chooser extensions, which both integrate into the system menu. 

If anyone wants to prepare a PR, I'd be happy to merge this, but I'd like to keep this functionality behind a config option.
